CODEOWNERS

```diff
@@ -0,0 +1,54 @@
+                          ################################################################################
+                          #          Each line is a file pattern followed by our owner/teams             #
+                          ################################################################################
+
+# This is a comment and will be ignored.
+
+# Global owner for everything in this repo
+* @rt4914
+
+# Owners/teams for backend (Ruby on Rails)
```

Comment:
Remove everyone below this, we just need the global owner, when required I will add things.

Reply:
I am on it